catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -548,8 +552,17 @@ private boolean validateFilePath(File resourceFilePath) throws IOException {
         return false;
     }
 
-    protected WebClient getWebClient(String uri) {
-        WebClient client = WebClient.create(uri);
+    protected WebClient getWebClient(String uri, Map<String, Serializable> properties) {
+        WebClient client;
+        if (properties.get(USERNAME) != null && properties.get(PASSWORD) != null) {
```

Comment:
(out of scope for this pr, but for the record) It would be a nice future improvement  to extract out the username/password string as an exported constant so that these changes and the previous change (https://codice.atlassian.net/browse/DDF-2547) wouldn't both have copies of the same two variables.